packages/wdio-cucumber-framework/README.md

```diff
@@ -141,6 +104,26 @@ Only retries the features or scenarios with tags matching the expression (repeat
 
 Type: `RegExp`
 
+### tagsInTitle
+Add cucumber tags to feature or scenario name
+
+Type: `Boolean`<br />
+Default: `false`
+
+### ignoreUndefinedDefinitions
+**Please note that this is a @wdio/cucumber-framework specific option and not recognized by cucumber-js itself**
+Treat undefined definitions as warnings.
+
+Type: `Boolean`<br />
+Default: `false`
+
+### failAmbiguousDefinitions
+**Please note that this is a @wdio/cucumber-framework specific option and not recognized by cucumber-js itself**
+Treat ambiguous definitions as errors.
+
+Type: `Boolean`<br />
+Default: `false`
```

Comment:
Is there a reason why the order was changed? Did anything change here?

Reply:
Not really, as these caps are specific to WebdriverIO, I've adjusted them accordingly.